measures/2019/measures-data.json

```diff
@@ -9890,7 +9890,7 @@
     "metricType": "boolean",
     "firstPerformanceYear": 2017,
     "lastPerformanceYear": null,
-    "weight": "high",
+    "weight": "medium",
```

Comment:
since the other change was unintended maybe it's worth checking on this one too? cc @sarahnw